### PR TITLE
Phemex: add auth error for "401 Failed to load API KEY."

### PR DESCRIPTION
**Pull request by phips28**
`{"code":"401","msg":"401 Failed to load API KEY."}`

### CODE REVIEW

File: js/src/phemex.js [hunk 1]
```diff
@@ -281,6 +281,7 @@ export default class phemex extends Exchange {
             'exceptions': {
                 'exact': {
                     // not documented
```
Comment (carlosmiei) [inline js/src/phemex.js:283]: @phips28 .JS files don't need to be committed they're automatically generated.  🙂

Comment (phips28) [reply js/src/phemex.js:283]: do you need a new PR? or does it work even if I have changed the JS file?

Comment (carlosmiei) [reply js/src/phemex.js]: I reverted it don't worry

**Comment by carlosmiei** [comment]
LGTM

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**